templates/home.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h2>Welcome to Mohsen Lab Receptor!</h2>
```

Comment:
name is spelled Mohsan. Also Mohsan is his first name. I think it would be Saeed Lab.
In either case, we should think about making this part of the config file as well - LAB_NAME = ""

Reply:
Added this as an Asana task in Deployment section.